recipes/libmbus/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "20200323":
+    sha256: 6607832790794c6e7b92a3c768868781f6dfacd48f80953e1653a0d48d03fdfe
+    url: https://github.com/gocarlos/libmbus/archive/a967850d1689e8b439b6501529b584c2696d2b17.zip
```

Comment:
Please use the upstream url: https://github.com/rscada/libmbus

I guess this is temporary to test [your cmake pull request](https://github.com/rscada/libmbus/pull/151)?

Reply:
Yes